space-cli/cmd/modules/operations/setup.go

```diff
@@ -143,7 +143,8 @@ func Setup(username, key, config, version, secret, clusterName string, dev bool,
 		"HOME_SECRETS_PATH=" + utils.GetMountTempSecretsDir(clusterName),
 		"HOSTS_FILE_PATH=" + utils.GetMountHostsFilePath(clusterName),
 		"ROUTING_FILE_PATH=" + "/routing-config.json",
-		"CLUSTER_ID=" + clusterID,
+		"CLUSTER_ID=" + clusterID,     // required for sending metrics
```

Comment:
I dont think this is required

Reply:
This flag is required, if you see the metric collection in runner. It uses this variable.

Gateway only gets an array of objects, aka values to be inserted. Each value contains the cluster id

Reply:
Got it